rest-fights/openapi-schema2.yml

```diff
@@ -0,0 +1,267 @@
+---
```

Comment:
You can just copy this as `openapi-schema.yml`, overwriting the existing one.

Reply:
Yes. I was trying to do some clever ordering while doing 1000 things in paralell and totaly mixed it up